boards/beagle/beagleconnect_freedom/beagleconnect_freedom.dts

```diff
@@ -187,7 +202,7 @@
 };
 
 &ieee802154 {
-	status = "okay";
+	status = "disabled";
```

Comment:
This looks like a breaking change. I understand the rationale behind it but this will break applications that had both drivers active before. (And it will obsolete configuration in all those applications that had it disabled in an overlay.) If I'm right, then you should add a comment about this in the breaking changes list of the release's change log.

Reply:
This should not be here. Let me remove it.